### PR TITLE
removed remount-sys

### PR DESCRIPTION
**Pull request by lmilbaum**

### CODE REVIEW

**Review by gbenhaim — APPROVED** [review]

File: centos-vm-libvirt/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM quay.io/pod_utils/systemd-libvirt:latest
+FROM quay.io/pod_utils/systemd-libvirt:1.0.3
```
Comment (ifireball) [inline centos-vm-libvirt/Dockerfile:1]: commit message should mention this change

**Review by ifireball — CHANGES_REQUESTED** [review]
Since I know we DO what to remount `/sys` please explain why remove it here

File: centos-vm-libvirt/Dockerfile (hunk 1 above)
Comment (lmilbaum) [reply centos-vm-libvirt/Dockerfile:1]: +1

**Comment by gbenhaim** [comment]
@ifireball We moved the remount to the base container 
https://github.com/no-name-yet/systemd-libvirt/blob/master/Dockerfile#L6

**Review by ifireball — APPROVED** [review]

**Pull request merged**